docs/attributes-registry/gen-ai.md

```diff
@@ -102,15 +102,22 @@ Datastore: A tool used by the agent to access and query structured or unstructur
 | `az.ai.openai` | Azure OpenAI | ![Development](https://img.shields.io/badge/-development-blue) |
 | `cohere` | Cohere | ![Development](https://img.shields.io/badge/-development-blue) |
 | `deepseek` | DeepSeek | ![Development](https://img.shields.io/badge/-development-blue) |
-| `gemini` | Gemini | ![Development](https://img.shields.io/badge/-development-blue) |
+| `gcp` | Any Google generative AI endpoint [6] | ![Development](https://img.shields.io/badge/-development-blue) |
```

Comment:
are we going to ever report abstract Google AI thing? Don't we know which model family are we talking to? 

If it's the case, let's still pick a name for it, something generic like `gcp.ai` or `gcp.gen_ai` like the one you had before

Reply:
OK to resolve?